src/api/vade_evan_api.rs

```diff
@@ -494,6 +496,107 @@ impl VadeEvan {
             .map_err(|err| err.into())
     }
 
+    /// Proposes to share a proof for a credential.
+    /// The proof proposal consists of the fields the verifier wants to be revealed per schema.
```

Comment:
should we rename this to 

```suggestion
    /// The proof proposal consists of the fields the prover wants to reveal per schema.
```

Reply:
adjusted ^^